addons/l10n_hu_edi/data/uom.uom.csv

```diff
@@ -0,0 +1,9 @@
+"id","l10n_hu_edi_code"
+"uom.product_uom_unit","PIECE"
```

Comment:
Thanks, I hadn't checked. The cubic meter was missing, and I've added it. FYI, here are the possible codes:
![image](https://github.com/odoo/odoo/assets/74787742/b373c116-e665-436d-aaf2-fd2e9d3c15bc)

Reply:
ITS BECAUSE THE VALUES THAT NEED TO BE PUT IN THE XML ARE CAPSLOCO
BESIDES, THE VALUES ARE NOT SHOWN TO THE USER
(the users just see `Piece`, `Kilogram` etc. because it's a selection field)